spec/fixtures/too_many_requests_error.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<IntuitResponse time="2016-12-07T15:27:09.527-08:00" xmlns="http://schema.intuit.com/finance/v3">
+  <Fault type="SERVICE">
+    <Error code="3001">
+      <Message>message=ThrottleExceeded; errorCode=003001; statusCode=429</Message>
```

Comment:
I actually thought it was odd that the message was *Throttle Exceeded* when in their blog post Intuit specifically said the message would be *Too Many Requests*. It really should be Too Many Requests if they are using a 429 response, at least according to this RFC: https://tools.ietf.org/html/rfc6585#section-4 and I'd argue that they are actually getting it wrong here. Of course, them getting it wrong doesn't mean we shouldn't just go with *Throttle Exceeded* since it's possible that they might just keep it wrong until the next version of the API is released.

Note that they are also deprecating the 503, which was a fairly unhelpful code to use for this in the first place since it actually should have been returning a *Service Unavailable* message and not *Throttle Exceeded*. I also think that you should probably be handling a 503 differently than a 429 - 503 should mean that the service is unavailable and you should retry using something like an exponential backoff, whereas for a 429 you could just retry a minute later, but that's just my opinion on that. On the other hand, they just seem to start redirecting with 403 responses when the API is actually unavailable so who knows. *sigh*

Happy to discuss this further. I'm actually also curious if anybody here has been receiving 429's yet in production. I haven't seen any yet... still seeing 503s.